### PR TITLE
[GEARPUMP-224] merge gearpump-daemon to gearpump-core

### PR DESCRIPTION
**Pull request by huafengw**

### CODE REVIEW

File: project/Pack.scala [hunk 1]
```diff
@@ -24,7 +24,6 @@ import xerial.sbt.Pack._
 object Pack extends sbt.Build {
   val daemonClassPath = Seq(
```
Comment (manuzhang) [inline project/Pack.scala]: shall we rename this classpath ?

Comment (huafengw) [inline project/Pack.scala]: suggestion?

Comment (manuzhang) [inline project/Pack.scala]: maybe keep it till we come up with a better name

**Comment by manuzhang** [comment]
LGTM, +1

**Pull request closed without merging**